src/com/seafile/seadroid2/transfer/TransferManager.java

```diff
@@ -83,6 +83,110 @@ public int addUploadTask(Account account, String repoID, String repoName,
         return task.getTaskID();
     }
 
+    private List<UploadTask> uploadWaitingList = Lists.newArrayList();
+    private List<UploadTask> uploadingList = Lists.newArrayList();
+    private static int mUploadingCount = 0;
```

Comment:
From the code I see `mUploadingCount` is always equal to `uploadingList.size()`, so I guess this variaible is not needed.